tests/func/experiments/test_show.py

```diff
@@ -505,6 +507,7 @@ def _get_rev_isotimestamp(rev):
     result1 = dvc.experiments.run(exp_stage.addressing, params=["foo=2"])
     rev1 = first(result1)
     ref_info1 = first(exp_refs_by_rev(scm, rev1))
+    time.sleep(1)
```

Comment:
does it makes sense to add a comment here?

Reply:
Only a temporary solution, I will create a new PR and replace this test with a serial of unit tests with man-made data to avoid the original problem.